index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PEGS 2023</title>
+  <link rel="icon" type="image/x-icon" href="/images/favicon.png">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+  <link rel="stylesheet" href="Style.css">
+  <script src="script.js" defer></script>
+</head>
+
+<body>
+  <div class="first-page">
+    <!-- headder -->
+    <header class="headder">
+      <a id="hamburger" class="hamburger text-dark">
+        <i class="glyphicon glyphicon-menu-hamburger mt-4"></i>
+      </a>
+      <div class="headding">
+        <div class="contact">
+          <i class="material-icons text-light">facebook</i>
+          <div class="text-light">English</div>
+          <div class="text-light">My Page</div>
+          <div class="text-light">Logout</div>
+        </div>
```

Comment:
- Please, kindly use a ul to group the list items as these are list items and you should use the ul to add them to improve the use of the HTML5 semantic tag.